bound/kt/src/test/kotlin/tbdex/sdk/httpclient/E2ePfiExemplarIntegrationTest.kt

```diff
@@ -0,0 +1,112 @@
+//            val balances = getBalances(pfiDidUri, bearerDid)
+//            println("Successfully retrieved balances")
+//            assertNotEquals(0, balances.size)
+
+            // create exchange
+            val rfq = Rfq(
+                bearerDid,
+                pfiDidUri,
+                bearerDid.did.uri,
+                CreateRfqData(
+                    offeringId = offerings[0].metadata.id,
+                    payin = CreateSelectedPayinMethod(
+                        "USD_LEDGER",
+                        null,
+                        "101"
+                    ),
+                    payout = CreateSelectedPayoutMethod(
+                        "MOMO_MPESA",
+                        ObjectMapper().readTree("""{
+                            "phoneNumber": "867-5309",
+                            "reason": "cause"
+                        }""")
+                    ),
+                    claims = listOf("eyJ0eXAiOiJKV1QiLCJhbGciOiJFZERTQSIsImtpZCI6ImRpZDpkaHQ6YzhkOWh1azduaG9tNG43emdybWE2cGp5Y3k2NzR1cmFhNHBvcDl1dXQ0MWdiOXd5OHNueSMwIn0.eyJ2YyI6eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSJdLCJ0eXBlIjpbIlZlcmlmaWFibGVDcmVkZW50aWFsIiwiU2FuY3Rpb25DcmVkZW50aWFsIl0sImlkIjoidXJuOnV1aWQ6ZjBkYWNlZmItNDVlNy00YWEyLTkxNDctMTZmYTBiYzc3ZTVjIiwiaXNzdWVyIjoiZGlkOmRodDpjOGQ5aHVrN25ob200bjd6Z3JtYTZwanljeTY3NHVyYWE0cG9wOXV1dDQxZ2I5d3k4c255IiwiaXNzdWFuY2VEYXRlIjoiMjAyNC0wNi0yNFQxNDoxNTozNVoiLCJjcmVkZW50aWFsU3ViamVjdCI6eyJpZCI6ImRpZDpkaHQ6MWZzNWhueHNndHhnZHI0d3pxaTM4Y25qNDZiMXdoaG45NG9qd282Nmc4aHNjNWJ0M2ZneSIsImJlZXAiOiJib29wIn19LCJuYmYiOjE3MTkyMzg1MzUsImp0aSI6InVybjp1dWlkOmYwZGFjZWZiLTQ1ZTctNGFhMi05MTQ3LTE2ZmEwYmM3N2U1YyIsImlzcyI6ImRpZDpkaHQ6YzhkOWh1azduaG9tNG43emdybWE2cGp5Y3k2NzR1cmFhNHBvcDl1dXQ0MWdiOXd5OHNueSIsInN1YiI6ImRpZDpkaHQ6MWZzNWhueHNndHhnZHI0d3pxaTM4Y25qNDZiMXdoaG45NG9qd282Nmc4aHNjNWJ0M2ZneSIsImlhdCI6MTcxOTIzODUzNX0.DvDFIl8BTuHRk7VkB82OhYpX0WzBb3BucvAqfXiS92QCiRokXCgQAsOwbbSODoDaFWbHG0BJmWM-eDPcCoucCw")
+                ),
+                "1.0", null
+            )
+            createExchange(rfq, null)
+            println("Successfully created exchange")
+
+            // get quote
+            var exchange = getExchange(pfiDidUri, bearerDid, rfq.metadata.exchangeId)
+            val quote = exchange.quote ?: throw Exception("Quote should not be null")
+            println("Successfully retrieved quote")
+
+            // submit order
+            submitOrder(Order(
+                bearerDid,
+                pfiDidUri,
+                bearerDid.did.uri,
+                quote.metadata.exchangeId,
+                "1.0", null
+            ))
+            println("Successfully submitted order")
+
+            // get order status and close
+            var count = 0
+            while (exchange.close == null) {
+                Thread.sleep(5000)
+                exchange = getExchange(pfiDidUri, bearerDid, quote.metadata.exchangeId)
```

Comment:
do we have to spin up a service or something? Is there some delay? why sleeping and retry?

Reply:
yeah this integration test expects a running instance of the [tbdex-pfi-exemplar](https://github.com/TBD54566975/tbd-examples/tree/main/javascript/tbdex-pfi-exemplar) and developers will have to update the `pfiDidUri`, the `didURi` and a couple other items for it to work on their end

I think we can make this better so it requires less setup, but that would probably require setting up env vars which are automatically set over in the [tbdex-pfi-exemplar](https://github.com/TBD54566975/tbd-examples/tree/main/javascript/tbdex-pfi-exemplar)